Reading List/Reading List/BookController.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  BookController.swift
+//  Reading List
+//
+//  Created by Dojo on 6/8/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import UIKit
+
+
+class BookController {
```

Comment:
Great job with the model controller overall, these things are not easy to grasp at first.